website/blog/2020-05-25-7.10.0.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "7.10.0 Released: Class Fields in preset-env, '#private in' checks and better React tree-shaking"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2020-05-25 0:00:00
+categories: announcements
+share_text: "Babel 7.10.0 Released"
+---
+
+We just released a new minor version of Babel!
+
+This 7.10 release includes:
+- Full support for the new Stage 1 proposal, `#prop in obj` checks for private fields [proposal](https://github.com/tc39/proposal-private-fields-in-in) (`#prop in obj`).
+- `@babel/preset-env` now compiles ES2015-style Unicode escapes (`\u{Babe1}`) to the equivalent legacy syntax (`\uDAAA\uDFE1`).
+- Two improvements to the [Optional Chaining](https://github.com/tc39/proposal-optional-chaining) operator (`?.`)
+- Parser support for the new Stage 1 [Module Attributes proposal](https://github.com/tc39/proposal-module-attributes) (`import a from "./a.json" with type: "json"`).
+- Better tree-shaking support for React code (i.e. `React.memo`)!
+- Setting up [RFCs](https://github.com/babel/rfcs) repo and [GitHub Discussions](https://github.com/babel/babel/discussions) pages!
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.10.0).
+
+<!-- truncate -->
+
+Alongside this Babel release, we are releasing the first experimental version of our new [polyfills compatibility architecture](https://github.com/babel/babel-polyfills) (see below for more details) thanks to [Nicolò](https://twitter.com/NicoloRibaudo) and some awesome folks in the community! We began discussions about this over a year ago in a [RFC issue](https://github.com/babel/babel/issues/10008) within the Babel repository.
+
+As an aside, we now have an official RFC process for discussing changes that significantly impact our users: please check it out over in the [`babel/rfcs`](https://github.com/babel/rfcs) repository! In addition, we've enabled [GitHub Discussions](https://github.com/babel/babel/discussions) on our repository if you have feedback or questions!
+
+If you or your company want to support Babel and the evolution of JavaScript, but aren't sure how, you can donate to us on our [Open Collective](https://opencollective.com/babel) and, better yet, work with us on the implementation of [new ECMAScript proposals](https://github.com/babel/proposals) directly! As a volunteer-driven project, we rely on the community's support to fund our efforts in supporting the wide range of JavaScript users. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss more!
+
+## Highlights
+
+### New features enabled by default
+
+Now that it has reached Stage 4, parsing for [`import.meta`](https://github.com/tc39/proposal-import-meta) is enabled by default. Please note that `@babel/preset-env` doesn't support transforming it, because what that object contains is up to the engine and is not defined in the ECMAScript specification.
```

Comment:
```suggestion
Now that it has reached Stage 4, _parsing_ for [`import.meta`](https://github.com/tc39/proposal-import-meta) is enabled by default. Please note that `@babel/preset-env` doesn't have any default support for _transforming_ it, because what that object contains is up to the engine and is not defined in the ECMAScript specification.
```
aside: if we know the target (node), it should be possible to transform some forms of it to some equivalent in some cases right (`import.meta.url`)?